db.go

```diff
@@ -573,7 +596,7 @@ func (db *DB) Close() error {
 	for _, table := range db.tables {
 		table.close()
 		if db.bucket != nil {
-			table.writeBlock(table.ActiveBlock())
+			table.writeBlock(context.TODO(), table.ActiveBlock())
```

Comment:
Should probably update to a real ctx that's threaded through

Reply:
This is a heavy lift because you have to pass it in through `columnStore.Close` and `db.Close` which seemed like a heavy refactor/out of scope for this PR. This is why I used `context.TODO`, to indicate that at some point in the future, we probably need to pass it an actual ctx.